Stratis.Bitcoin.Tests/BlockStore/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Stratis.Bitcoin.Tests.BlockStore.LoopTests
 {
+    /// <summary>
+    /// Base test class for all the BlockStoreLoop tests
+    /// </summary>
     public class BlockStoreLoopStepBaseTest
```

Comment:
SENTENCES rule - basically everywhere.
Also consider using `<see>` for BlockStoreLoop here (and everywhere where you are talking about existing class/object/variable/...) - this allows the name to be clicked through in IDE, which is useful.